guides/doc-Configuring_Load_Balancer/topics/Configuring_Capsule_Server_with_Default_SSL_Certificates_for_Load_Balancing_with_Puppet.adoc

```diff
@@ -68,21 +68,41 @@ root@_{smart-proxy-context}_ca.example.com_:__{smart-proxy-context}_ca.example.c
 --enable-foreman-proxy-plugin-remote-execution-ssh
 ----
 
-. On {SmartProxyServer}, generate Puppet certificates for all other {SmartProxyServer}s that you configure for load balancing, except this first system where you configure Puppet certificates signing:
+. On {SmartProxyServer}, generate Puppet certificates for all other {SmartProxyServer}s that you configure for load balancing, except the first system where you configure Puppet certificates signing:
+.. First, stop the Puppet server:
 +
 [options="nowrap", subs="+quotes,verbatim,attributes"]
 ----
-# puppet cert generate _{smartproxy-example-com}_ \
---dns_alt_names=_loadbalancer.example.com_
+# puppet resource service puppetserver ensure=stopped
 ----
 +
+.. Generate certificate for the {SmartProxyServer}:
++
+[options="nowrap", subs="+quotes,verbatim,attributes"]
+----
+# puppetserver ca generate --certname _{smartproxy-example-com}_ \
+--subject-alt-names _loadbalancer.example.com_ --ca-client
+----
++
+WARNING: The '--ca-client' option in the command must only be used for server load balancers. Using the '--ca-client' option to create certificates for regular nodes can create security issues.
++
 This command creates the following files on the system where you configure {SmartProxyServer} to sign Puppet certificates:
 +
 * `/etc/puppetlabs/puppet/ssl/certs/ca.pem`
 * `/etc/puppetlabs/puppet/ssl/certs/_{smartproxy-example-com}_.pem`
 * `/etc/puppetlabs/puppet/ssl/private_keys/_{smartproxy-example-com}_.pem`
 * `/etc/puppetlabs/puppet/ssl/public_keys/_{smartproxy-example-com}_.pem`
 
+.. Resume the Puppet server:
++
+[options="nowrap", subs="+quotes,verbatim,attributes"]
+----
+# puppet resource service puppetserver ensure=running
+----
++
+
+
+
```

Comment:
a redundant `+` and extra new lines here

Reply:
Fixed